spec/jobs/address/check_validity_job_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2012-2020, CVP Schweiz. This file is part of
+#  hitobito_cvp and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_cvp.
+
+require 'spec_helper'
+
+describe Address::CheckValidityJob do
```

Comment:
Bitte anpassen, sodass wir die wesentlichen Konzepte testen (person wird getagged, emails werden enqueued). Dafür reichen dann einfach assertions (1 tag angelegt, 2 emails enqueued), vielleicht mit `ActiveJob::TestHelper` (siehe ImpersonationsControllerSpec)  
 
Das testen der Details von `Address::ValidationChecksMailer` and `Contactable::AddressValidator` sollte in den entsprechenden unit tests passieren.